plugins/reach/lib/model/assetHandler.php

```diff
@@ -8,6 +8,16 @@ public static function shouldAddEntryVendorTask($object, $vendorCatalogItem): bo
 			return true;
 	}
 
+	public static function shouldAddEntryVendorTaskByObject($object, $vendorCatalogItem, $reachProfile) : bool
+	{
+		if (!$vendorCatalogItem->isAssetSupported($object))
+		{
+			KalturaLog::log("assets are not supported");
```

Comment:
Print out is useless, unless you give some info on the supported / not supported asset, maybe in case of not supported, do the print inside isAssetSupported or throw so the print can have some info about "what was not supported"